src/tests/ftest/osa/osa_dmg_negative_test.yaml

```diff
@@ -0,0 +1,69 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+test_clients:
+    - client-D
+# If we define the server under test_servers, launch.py will convert it to the
+# actual server name passed into --test_servers. If above test_servers is hosts,
+# it'll be used as one of the servers at test startup time, so use something
+# other than hosts.
+extra_servers:
+  test_servers:
+    - server-C
+timeout: 1000
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["0000:81:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,md,rebuild
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31516
+      log_file: daos_server1.log
+      bdev_class: nvme
+      bdev_list: ["0000:da:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,md,rebuild
+pool:
+  mode: 146
+  name: daos_server
+  scm_size: 6000000000
+  nvme_size: 54000000000
+  svcn: 4
+  control_method: dmg
+container:
+  properties:
+    enable_checksum: True
+dmg_cmds:
+  dmg_cmd_test:
```

Comment:
Just my preference, but you used test_seq in the code, so it would be easier to read if you use it here too.

Reply:
Done.